middleware/middleware-1.3.24.md

```diff
@@ -0,0 +1,64 @@
+---
+slug: /release-notes/middleware/1.3.24
+title: Version 1.3.24
+---
+
+# fiskaltrust.Middleware 1.3.24 (Germany)
+_October 15, 2021_
+
+In this quality-of-life release of the Middleware, we resolved several issues that were occurring for segments of our users, and introduced multiple stability improvements to further improve the behavior of the Middleware during implementation and runtime.
+
+:::caution
+
+Version 1.3 of the Middleware is meant for the German market only, customers in Austria and France should continue to use version 1.2. We will unify these experiences in an upcoming version.
+
+:::
+
+## Bug fix: FCC update failed when service folder is set to a different drive
+Due to an issue in the recently added update process for the Fiskal Cloud Connector (the 3rd part dependency used by the _Swissbit Cloud_ and _Deutsche Fiskal_ SCUs), the FCC update was not successful when switching to SCU package version 1.3.23. FCC installations were not damaged by this failure, as it occurred before the actual update took place. The issue has been resolved, and the FCC will be properly updated when upgrading to SCU versions >= 1.3.24.
+
+## Bug fix: Configuration upload failed when a Queue was put into a different Cashbox
+When a Queue was put into a different Cashbox (e.g. when large Cashboxes are split up due to performance reasons), the Middleware mistook this change as a forbidden configuration update, which lead to blocked uploads of the configuration data (e.g. the Queue's status). Relevant receipt data was not affected by this and properly uploaded, but some convenience features in our Portal were blocked by this issue (e.g. viewing the Queue state). Due to our retry mechanisms, the configuration data will be uploaded as soon as the Queue version is updated and rolled out.
+
+:::tip
+
+Please note that installing an already running Cashbox on a different machine is not possible without further steps. The scenario described above only applies to situations where the Queue was put into another Cashbox that is then running on the _same_ machine.
+
+:::
+
+## Bug fix: TAR export via the API failed on some Linux distributions
+We've received reports about an issue that lead to failing TAR exports on some Linux distributions due to a missing dependency. The issue has been resolved.
+
+## Bug fix: Large numbers were incorrectly formatted in TSE process data
+We've resolved an issue in the Middleware that could lead to incorrectly formatted TSE process data in some rare cases when the overall amount of a single receipt exceeded € 1000. Due to this error, a thousand separator was inserted, although this is not provided for in the DSFinV-K. 
```

Comment:
Provided for is probably wrong in this case

```suggestion
We've resolved an issue in the Middleware that could lead to incorrectly formatted TSE process data in some rare cases when the overall amount of a single receipt exceeded € 1000. Due to this error, a thousand separator was inserted, although this is not necessary in the DSFinV-K. 
```

Reply:
I was also not familiar with this, but "not provided for" means "nicht vorgesehen" :D thanks for the suggestion though.